backend/src/app/entities/sites.entity.ts

```diff
@@ -216,7 +216,7 @@ export class Sites extends ChangeAuditEntity {
     precision: 4,
     scale: 2,
   })
-  latSeconds: string | null;
+  latSeconds: number | null;
```

Comment:
I think if we are changing from string, we should have type decimal instead of number. Please check it again.